test/unitTest/java/org/thoughtcrime/securesms/NicknameMocks.java

```diff
@@ -0,0 +1,61 @@
+package org.thoughtcrime.securesms;
+
+import android.content.Context;
+
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.thoughtcrime.securesms.database.DatabaseFactory;
+import org.thoughtcrime.securesms.database.RecipientDatabase;
+import org.thoughtcrime.securesms.recipients.Recipient;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockingDetails;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({DatabaseFactory.class})
+public class NicknameMocks extends BaseUnitTest {
+    protected DatabaseFactory   databaseFactory;
+    protected Recipient         recipient;
+    protected Recipient         recipient2;
```

Comment:
this one is not used anywhere.

Reply:
what do you mean?
they are, maybe not recipient2

Reply:
`recipient2` is not used anywhere